tests/system.py

```diff
@@ -736,6 +736,65 @@ def test_load_table_from_dataframe_w_automatic_schema(self):
         )
         self.assertEqual(table.num_rows, 3)
 
+    @unittest.skipIf(
+        pandas is None or pandas.__version__ < "1.0.1",
+        "Only `pandas version >=1.0.1` supports",
```

Comment:
```suggestion
        "Only `pandas version >=1.0.1` supported",
```
Seems like more correct? (disclaimer: not a native speaker)